src/instructlab/sdg/datamixing.py

```diff
@@ -325,6 +327,76 @@ def _conv_pretrain(rec):
     return rec
 
 
+def _load_auxiliary_instructions():
+    """
+    Load the auxiliary instructions yaml from disk, returning the loaded
+    object if found or None if the instructions yaml file does not exist.
+    """
+    auxilary_path = resources.files(__package__).joinpath(
+        "configs/knowledge/auxiliary_instructions.yaml"
```

Comment:
This is mirroring where these files are already placed by some of our downstream users, so it could be moved but we'll need to coordinate with them to ensure downstream auxiliary instructions gets moved as well.

Reply:
Yes please

Reply:
Naming is hard for me, but I took a first stab at this by moving this from `configs/` to `instructions/auxiliary_knowledge.yaml`.